java/vector/src/test/java/org/apache/arrow/vector/TestValueVector.java

```diff
@@ -1394,6 +1394,35 @@ public void testFillEmptiesNotOverfill() {
     }
   }
 
+  @Test
+  public void testSetSafeWithArrowBuf() {
```

Comment:
maybe add to the title doesntAllocateExcessiveMemory

Reply:
done